tests/conftest.py

```diff
@@ -18,7 +21,28 @@ class User:
     email = attr.ib(type=str)
 
 
-@pytest.fixture(name='org')
+@pytest.fixture(autouse=True)
+def _verify_url(request, server_url, user, org):
+    """Verifies the base URL.
+    
+    This will ping the base url until it returns a 200.
+    """
+    verify = request.config.option.verify_server_url
+    if server_url and verify:
+        session = requests.Session()
+        retries = Retry(backoff_factor=0.1,
+                        status_forcelist=[500, 502, 503, 504])
+        session.mount(server_url, HTTPAdapter(max_retries=retries))
+        session.get(server_url, verify=False)
```

Comment:
What happens if the server ping is unsuccessful? Could it be blocking the test run indefinitely? It would be better to raise an error and abort the test run with a descriptive message.

If ``server_url`` is not set, we don't perform any pings. I'd rather raise an error from ``fixture_server_url`` than failing silently.

Reply:
It will fail with an error saying that the URL could not be loaded. It will be an ```HTTPAdapter``` error.

Reply:
If we don't specify a URL pytest will fail with a selenium error saying bad URL, I don't think we need to check for that specifically.